<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="mpl_widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>681</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wToolbar" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>541</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>561</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="standardViewBtn">
         <property name="text">
          <string>Standard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hierarchyViewBtn">
         <property name="text">
          <string>Hierarchy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tracerViewBtn">
         <property name="text">
          <string>Tracer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exampleBtn">
         <property name="text">
          <string>Example</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="setAddressBtn">
         <property name="text">
          <string>Set Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="setAddressTxt"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
